velox/docs/functions/spark/json.rst

```diff
@@ -0,0 +1,13 @@
+==============
+Json Functions
+==============
+
+.. spark:function:: json_object_keys(jsonString) -> array(string)
+
+    Returns all the keys of the outermost JSON object as an array if a valid JSON object is given. If it is any other valid JSON string, an invalid JSON string or an empty string, the function returns null. ::
+
+        SELECT json_object_keys('{}'); -- []
+        SELECT json_object_keys('{"name": "Alice", "age": 5, "id": "001"}'); -- ['name', 'age', 'id']
+        SELECT json_object_keys(''); -- NULL
+        SELECT json_object_keys(1); -- NULL
+        SELECT json_object_keys('hello'); -- NULL
```

Comment:
I assume `1` represents invalid json string, and `'hello'` represents a valid json string but not json objects. But it seems `'hello'` is also an invalid json string. Perhaps we need "hello" here. Could you confirm? Thanks.

> spark.sql("SELECT schema_of_json('hello');").show(false)
com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'hello': was expecting (JSON String, Number (or 'NaN'/'INF'/'+INF'), Array, Object or token 'null', 'true' or 'false')

> spark.sql("SELECT schema_of_json('\"hello\"');").show(false)
+-----------------------+
|schema_of_json("hello")|
+-----------------------+
|STRING                 |
+-----------------------+

Reply:
I get your point, there are three caases in which a NULL result will be produced:
1. an invalid JSON string